src/backend/api/mirrors_update.py

```diff
@@ -421,7 +432,7 @@ async def update_mirror_in_db(
     )
     db_session.add(mirror_to_create)
     logger.debug(
-        'Mirror "%s" is addded',
+        'Mirror "%s" is added',
```

Comment:
What about formatting via f-string?

Reply:
Never don't use formatting/f-string for logging, because
1. It's slow way
2. logging has itself formatting
3. Sentry won't group events by a log message, because each of them will be unique due to a formatted variable